### PR TITLE
fix: problems with getting enviroment variables from parent folder s…

### PR DESCRIPTION
**Pull request by KajaKoren**
Problems with getting getting variables from the .envrc file from nrf-asset-tracker folder folder to web-app folder, should be fixed now.

### CODE REVIEW

File: docs/aws/GettingStarted/Webapp.rst [hunk 1]
```diff
@@ -48,6 +48,14 @@ You need to configure the web application to be able to run it with your account
 
        echo "export PUBLIC_VERSION=\"`git describe --tags $(git rev-list --tags --max-count=1)`\"" >> .envrc
 
+#. In order to get the variables from the :file:`.envrc` file in our parent folder (:file:`nrf-asset-tracker`) we also need to add the following:
```
Comment (peknis) [inline docs/aws/GettingStarted/Webapp.rst]:
```suggestion
#. To get the variables from the :file:`.envrc` file in the parent folder (:file:`nrf-asset-tracker`), you also need to add the following entry in the :file:`.envrc` file of the current :file:`web-app` folder:
```

File: docs/aws/GettingStarted/Webapp.rst [hunk 2]
```diff
@@ -48,6 +48,14 @@ You need to configure the web application to be able to run it with your account
...
+
+   .. code-block:: bash
+
+      source_up
+
+   in the :file:`.envrc` in our current folder, :file:`web-app`.
```
Comment (peknis) [inline docs/aws/GettingStarted/Webapp.rst]:
```suggestion
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: docs/aws/GettingStarted/Webapp.rst [hunk 3]
```diff
@@ -48,6 +48,13 @@ You need to configure the web application to be able to run it with your account
 
        echo "export PUBLIC_VERSION=\"`git describe --tags $(git rev-list --tags --max-count=1)`\"" >> .envrc
 
+#. To get the variables from the :file:`.envrc` file in the parent folder (:file:`nrf-asset-tracker`), you also need to add the following entry in the :file:`.envrc` file of the current :file:`web-app` folder:
```
Comment (coderbyheart) [inline docs/aws/GettingStarted/Webapp.rst]:
```suggestion
#. To get the variables from the :file:`.envrc` file in the parent folder (:file:`~/nrf-asset-tracker`), you also need to add the following entry in the :file:`.envrc` file of the current :file:`web-app` folder:
```

**Review by coderbyheart — APPROVED** [review]

**Review by peknis — APPROVED** [review]

File: docs/aws/GettingStarted/Webapp.rst [hunk 4]
```diff
@@ -48,6 +48,13 @@ You need to configure the web application to be able to run it with your account
 
        echo "export PUBLIC_VERSION=\"`git describe --tags $(git rev-list --tags --max-count=1)`\"" >> .envrc
 
+#. To get the variables from the :file:`.envrc` file in the parent folder (:file:`~/nrf-asset-tracker`), you also need to add the following entry in the :file:`.envrc` file of the current :file:`web-app` folder:
+
+   .. code-block:: bash
+
+      source_up
+
```
Comment (peknis) [inline docs/aws/GettingStarted/Webapp.rst:56]: Just a nit: You might want to remove an extra empty line here.

**Review by wiba-nordic — APPROVED** [review]

**Comment by coderbyheart** [comment]
Thanks @Semikaja!

**Pull request merged**